### PR TITLE
disabled a visual unit test which fails most of the time so travis su…

### PR DESCRIPTION
**Pull request by pchien**
…ccess/fail will be more useful. increasing timeout for test that failed on timeout on travis. changed .travis.yml to make it run properly and also run unit tests and pod lib lint (mostly from swift .travis.yml)

### CODE REVIEW

File: HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1573,6 +1574,7 @@
 				MIXPANEL_ERROR = "";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.mixpanel.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 2.3;
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj]: is there a reason why we have a swift version here now

File: HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1614,6 +1616,7 @@
...
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj]: and here

File: HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1756,6 +1759,7 @@
...
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj]: and here

File: HelloMixpanel/HelloMixpanel.xcodeproj/xcshareddata/xcschemes/[iOS] HelloMixpanel.xcscheme [hunk 4]
```diff
@@ -54,6 +54,9 @@
             </BuildableReference>
             <SkippedTests>
                <Test
+                  Identifier = "MPNotificationTests/testVisualNotifications">
```
Comment (yarneo) [inline HelloMixpanel/HelloMixpanel.xcodeproj/xcshareddata/xcschemes/[iOS] HelloMixpanel.xcscheme]: would this mean that we would be able to run it personally though?

Comment (pchien) [inline HelloMixpanel/HelloMixpanel.xcodeproj/xcshareddata/xcschemes/[iOS] HelloMixpanel.xcscheme]: yeah we can turn it on and off locally whenever we want

**Comment by yarneo** [comment]
LGTM!

**Pull request merged**